types/prettier/parser-angular.d.ts

```diff
@@ -0,0 +1,5 @@
+import { Plugin } from './';
+
+declare const plugin: Plugin;
+export as namespace plugin;
```

Comment:
It's actually exported as `prettierPlugin[pluginName]`, see https://prettier.io/docs/en/browser.html#global.